libs/providers/flagd/src/lib/configuration.ts

```diff
@@ -53,6 +88,15 @@ const getEnvVarConfig = (): Partial<Config> => ({
   ...(process.env[ENV_VAR.FLAGD_SOCKET_PATH] && {
     socketPath: process.env[ENV_VAR.FLAGD_SOCKET_PATH],
   }),
+  ...(process.env[ENV_VAR.FLAGD_CACHE] && (process.env[ENV_VAR.FLAGD_CACHE] === 'lru' || process.env[ENV_VAR.FLAGD_CACHE] === 'disabled') && {
```

Comment:
Do you need this part? `process.env[ENV_VAR.FLAGD_CACHE] &&`

Reply:
no

Reply:
fixed